src/openms/source/CHEMISTRY/sources.cmake

```diff
@@ -29,12 +29,12 @@ RNaseDigestion.cpp
 Ribonucleotide.cpp
 RibonucleotideDB.cpp
 SpectrumAnnotator.cpp
-TheoreticalSpectrumGeneratorXLMS.cpp
 SimpleTSGXLMS.cpp
 SvmTheoreticalSpectrumGenerator.cpp
 SvmTheoreticalSpectrumGeneratorTrainer.cpp
 SvmTheoreticalSpectrumGeneratorSet.cpp
 TheoreticalSpectrumGenerator.cpp
+TheoreticalSpectrumGeneratorNA.cpp
```

Comment:
`NA` has a weird connotation.
Would `NucAcid` be an option?

Reply:
I understand your objection, but I don't like the "NucAcid" suffix. How about "NucleicAcidSpectrumGenerator"?

Reply:
I've renamed the class.